Mathlib/GroupTheory/Submonoid/Operations.lean

```diff
@@ -79,39 +79,46 @@ section
 
 /-- Submonoids of monoid `M` are isomorphic to additive submonoids of `additive M`. -/
 @[simps]
-def Submonoid.toAddSubmonoid : Submonoid M ≃o AddSubmonoid (Additive M)
-    where
+def Submonoid.toAddSubmonoid :
+    Submonoid M ≃o
+      AddSubmonoid
+        (Additive
+          M) where
   toFun S :=
     { carrier := Additive.toMul ⁻¹' S
       zero_mem' := S.one_mem'
-      add_mem' := fun _ _ => S.mul_mem' }
+      add_mem' := fun ha hb => S.mul_mem' ha hb }
   invFun S :=
     { carrier := Additive.ofMul ⁻¹' S
       one_mem' := S.zero_mem'
-      mul_mem' := fun _ _ => S.add_mem' }
-  left_inv x := by cases x <;> rfl
-  right_inv x := by cases x <;> rfl
-  map_rel_iff' a b := Iff.rfl
+      mul_mem' := fun ha hb => S.add_mem' ha hb}
+  left_inv x := by cases x; rfl
+  right_inv x := by cases x; rfl
+  map_rel_iff' := Iff.rfl
 #align submonoid.to_add_submonoid Submonoid.toAddSubmonoid
 
-/-- Additive submonoids of an additive monoid `additive M` are isomorphic to submonoids of `M`. -/
+/-- Additive submonoids of an additive monoid `Additive M` are isomorphic to submonoids of `M`. -/
 abbrev AddSubmonoid.toSubmonoid' : AddSubmonoid (Additive M) ≃o Submonoid M :=
   Submonoid.toAddSubmonoid.symm
 #align add_submonoid.to_submonoid' AddSubmonoid.toSubmonoid'
 
 theorem Submonoid.to_add_submonoid_closure (S : Set M) :
-    (Submonoid.closure S).toAddSubmonoid = AddSubmonoid.closure (Additive.toMul ⁻¹' S) :=
-  le_antisymm
-    (Submonoid.toAddSubmonoid.le_symm_apply.1 <| Submonoid.closure_le.2 AddSubmonoid.subset_closure)
-    (AddSubmonoid.closure_le.2 Submonoid.subset_closure)
-#align submonoid.to_add_submonoid_closure Submonoid.to_add_submonoid_closure
+    Submonoid.toAddSubmonoid (Submonoid.closure S)
+      = AddSubmonoid.closure (Additive.toMul ⁻¹' S) := by
+    apply le_antisymm
+      (Submonoid.toAddSubmonoid.le_symm_apply.mp (Submonoid.closure_le.mpr _))
+      (AddSubmonoid.closure_le.mpr _)
+    exact @AddSubmonoid.subset_closure (Additive M) _ S
+    exact @Submonoid.subset_closure M _ S
```

Comment:
```suggestion
  apply le_antisymm
      (Submonoid.toAddSubmonoid.le_symm_apply.mp (Submonoid.closure_le.mpr _))
      (AddSubmonoid.closure_le.mpr _)
  exact @AddSubmonoid.subset_closure (Additive M) _ S
  exact @Submonoid.subset_closure M _ S
```
If I understand the style guide corrrectly: indent lines 2+ in the theorem/definition statement by four spaces, but indent the lines of a tactic _proof_ by 2 spaces.